apps/odyssey-stats/src/app.tsx

```diff
@@ -61,6 +61,28 @@ async function AppBoot() {
 	setStore( store as Store & WithAddReducer );
 	setupContextMiddleware( store, queryClient );
 
+	// Initialize site data early in the app boot process.
+	await new Promise< void >( ( resolve ) => {
+		const mockContext = {
+			store,
+			query: {},
+			params: {},
+			path: '',
+			pathname: '',
+			title: '',
+			save: () => {},
+			pushState: () => {},
+			handled: false,
+			canonicalPath: '',
+			originalPath: '',
+			state: initialState,
```

Comment:
```suggestion
			state: '',
```
This state does not seem to correspond to the `initialState` that has been injected into the store.